pages/myPlants.js

```diff
@@ -0,0 +1,34 @@
+import Footer from "@/components/Footer";
+import PlantList from "@/components/PlantList";
+import styled from "styled-components";
+
+export default function MyPlantsPage({ plants, toggleFavourite }) {
+  const favouritePlants = plants
+    ? plants.filter((plant) => plant.isFavourite)
+    : [];
+
+  return (
+    <main>
+      <StyledHeader>My Plants</StyledHeader>
+      {favouritePlants.length > 0 ? (
+        <PlantList plants={favouritePlants} toggleFavourite={toggleFavourite} />
+      ) : (
+        <>
+          <StyledParagraph>
+            No favourite plants found. Please select some favourite plants.
+          </StyledParagraph>
+        </>
+      )}
+      <Footer />
```

Comment:
Remove the `<Footer/>` component from here. You have already implemented it in the Layout component.